pom.xml

```diff
@@ -86,6 +93,12 @@
         <version>1.2</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.jenkins-ci.plugins.workflow</groupId>
+        <artifactId>workflow-support</artifactId>
+        <version>3.0</version>
```

Comment:
Updated to 3.0 to avoid https://issues.jenkins-ci.org/browse/JENKINS-51998.

And moved to top to avoid `RequireUpperBoundDeps`:
```
Require upper bound dependencies error for org.jenkins-ci.plugins:script-security:1.26 paths to dependency are:
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-cps:2.32
    +-org.jenkins-ci.plugins:script-security:1.26
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-cps:2.32
    +-org.jenkins-ci.plugins:script-security:1.26
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-durable-task-step:2.13
    +-org.jenkins-ci.plugins:script-security:1.25
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-support:3.0
    +-org.jenkins-ci.plugins:script-security:1.39
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-multibranch:2.10
    +-org.jenkins-ci.plugins:script-security:1.23
,
Require upper bound dependencies error for org.jenkins-ci.plugins:scm-api:2.0.8 paths to dependency are:
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins:scm-api:2.0.8
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-cps:2.32
    +-org.jenkins-ci.plugins:scm-api:2.0.7
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-cps:2.32
    +-org.jenkins-ci.plugins:scm-api:2.0.7
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-support:3.0
    +-org.jenkins-ci.plugins:scm-api:2.2.6
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-api:2.28
    +-org.jenkins-ci.plugins:scm-api:2.0.8
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-api:2.28
    +-org.jenkins-ci.plugins:scm-api:2.0.8
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-multibranch:2.10
    +-org.jenkins-ci.plugins:scm-api:2.0.1
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-multibranch:2.10
    +-org.jenkins-ci.plugins:branch-api:2.0.0
      +-org.jenkins-ci.plugins:scm-api:2.0.1
```

Reply:
Better to take 3.2 due to known Pipeline issues

Reply:
3.2 requires updates for more dependencies (e.g struct) and I decided to use 3.0 for this time, which is the least version to have tests run on Java11.